src/DynamoCore/Search/SearchElements/ZeroTouchSearchElement.cs

```diff
@@ -136,4 +136,51 @@ private string GetIconName()
             }
         }
     }
+
+    internal class ReflectionZeroTouhSearchElement : NodeSearchElement
+    {
+        private readonly ReflectionFunctionDescriptor functionDescriptor;
+        private readonly string fullname;
+        public override string CreationName { get { return functionDescriptor != null ? functionDescriptor.MangledName : this.Name; } }
+
+        public override string FullName { get { return fullname; } }
+
+        internal ReflectionFunctionDescriptor Descriptor => functionDescriptor;
+
+        public ReflectionZeroTouhSearchElement(ReflectionFunctionDescriptor functionDescriptor)
+        {
+            this.functionDescriptor = functionDescriptor;
+
+            Name = functionDescriptor.UserFriendlyName;
+
+            if (functionDescriptor.IsOverloaded)
+            {
+                var parameters = new StringBuilder();
+                parameters.Append("(");
+                parameters.Append(String.Join(", ", functionDescriptor.Parameters.Select(x => x.Name)));
+                parameters.Append(")");
+
+                Parameters = parameters.ToString();
+            }
+
+            FullCategoryName = functionDescriptor.Category;
+            Description = functionDescriptor.Description;
+            Assembly = functionDescriptor.Assembly;
+
+            //Create full name including assembly name
+            fullname = string.Format("{0}.{1}", System.IO.Path.GetFileNameWithoutExtension(Assembly), functionDescriptor.QualifiedName);
+            ElementType = ElementTypes.ZeroTouch;
+
+            if (functionDescriptor.IsBuiltIn)
+                ElementType |= ElementTypes.BuiltIn;
+
+            if (functionDescriptor.IsPackageMember)
+                ElementType |= ElementTypes.Packaged;
+        }
+
+        protected override NodeModel ConstructNewNodeModel()
+        {
+            throw new NotImplementedException();
```

Comment:
If this doesn have an implementation might be better to to `base.ConstructNewNodeModel()` or even not overriding the method?

Reply:
the ConstructNewNodeModel is marked abstract in the base, so i have to implement it here. However, as this is meant to be used in ReflectionOnly we cant create anything from these types as they are only loaded into the `MetaDataLoadContext`.